src/main/kotlin/com/openlattice/search/renderers/CAREIssueAlertEmailRenderer.kt

```diff
@@ -0,0 +1,175 @@
+
+            val firstName = (combinedEntity[FIRST_NAME_FQN] ?: emptySet()).joinToString("/")
+            val lastName = (combinedEntity[LAST_NAME_FQN] ?: emptySet()).joinToString("/")
+            val middleName = (combinedEntity[MIDDLE_NAME_FQN] ?: emptySet()).joinToString("/")
+
+            return mapOf("formattedName" to "$lastName, $firstName $middleName")
+        }
+
+        private fun getReporterDetails(
+                neighbors: List<NeighborEntityDetails>, staffEntitySetId: UUID, reporterEntitySetId: UUID
+        ): Map<String, String> {
+            val combinedEntity = getCombinedNeighbors(neighbors, staffEntitySetId, Optional.of(reporterEntitySetId))
+
+            return mapOf("reporter" to (combinedEntity[PERSON_ID_FQN] ?: emptySet()).joinToString(", "))
+        }
+
+        private fun getAssigneeDetails(
+                neighbors: List<NeighborEntityDetails>, staffEntitySetId: UUID, assigneeEntitySetId: UUID
+        ): Map<String, String> {
+            val combinedEntity = getCombinedNeighbors(neighbors, staffEntitySetId, Optional.of(assigneeEntitySetId))
+
+            return mapOf("assignee" to (combinedEntity[PERSON_ID_FQN] ?: emptySet()).joinToString(", "))
+        }
+
+        private fun getIssueDetails(
+                issue: Map<FullQualifiedName, Set<Any>>, timeZone: MessageFormatters.TimeZones
+        ): Map<String, String> {
+
+            val completedDateTime = (issue[COMPLETED_DATE_TIME_FQN] ?: emptySet()).map { OffsetDateTime.parse(it.toString()) }
+            val createdDate = completedDateTime.joinToString(", ") { MessageFormatters.formatDate(it, timeZone) }
+            val createdTime = completedDateTime.joinToString(", ") { MessageFormatters.formatTime(it, timeZone) }
+
+            val updatedDateTime = (issue[ENTRY_UPDATED_FQN] ?: emptySet()).map { OffsetDateTime.parse(it.toString()) }
+            val updatedDate = updatedDateTime.joinToString(", ") { MessageFormatters.formatDate(it, timeZone) }
+            val updatedTime = updatedDateTime.joinToString(", ") { MessageFormatters.formatTime(it, timeZone) }
+            val issueId = (issue[EdmConstants.ID_FQN] ?: emptySet()).joinToString(", ")
+
+            return mapOf(
+                    "createdDateTime" to "$createdDate $createdTime",
+                    "updatedDateTime" to "$updatedDate $updatedTime",
+                    "status" to (issue[STATUS_FQN] ?: emptySet()).joinToString(", "),
+                    "title" to (issue[TITLE_FQN] ?: emptySet()).joinToString(", "),
+                    "description" to (issue[DESCRIPTION_FQN] ?: emptySet()).joinToString(", "),
+                    "category" to (issue[CATEGORY_FQN] ?: emptySet()).joinToString(", "),
+                    "priority" to (issue[PRIORITY_FQN] ?: emptySet()).joinToString(", "),
+                    "viewIssueTag" to "<a href=\"http://openlattice.com/issue/$issueId\">View issue</a>"
```

Comment:
should this be `/care/#/issue/etc`? 

I also just want to double check that `issueId` is not a user input string.

Reply:
you're right. issueId comes from the issue entityKeyId